tests/unit_tests/test_rating_service.py

```diff
@@ -205,7 +205,7 @@ async def test_get_new_player_rating_created(semiinitialized_service):
     tables, a new rating entry should be created.
     """
     service = semiinitialized_service
-    player_id = 999
+    player_id = 300
```

Comment:
Ah, I guess this test was only working before since foreign keys weren't properly enforced?

Reply:
Yea, this test and the other one too